runtime/Java/pom.xml

```diff
@@ -1,103 +1,91 @@
-		</developer>
-
-		<developer>
-			<name>Jim Idle</name>
-			<email>jimi@idle.ws</email>
-			<url>http://www.linkedin.com/in/jimidle</url>
-			<roles>
-				<role>Developer - Maven Plugin</role>
-			</roles>
-		</developer>
-	</developers>
-
-	<mailingLists>
-		<mailingList>
-			<name>antlr-discussion</name>
-			<archive>https://groups.google.com/forum/?fromgroups#!forum/antlr-discussion</archive>
-		</mailingList>
-	</mailingLists>
-
-	<issueManagement>
-		<system>GitHub Issues</system>
-		<url>https://github.com/antlr/antlr4/issues</url>
-	</issueManagement>
-
-	<scm>
-		<url>https://github.com/antlr/antlr4/tree/master</url>
-		<connection>scm:git:git://github.com/antlr/antlr4.git</connection>
-		<developerConnection>scm:git:git@github.com:antlr/antlr4.git</developerConnection>
-		<tag>HEAD</tag>
-	</scm>
-
-	<distributionManagement>
-		<snapshotRepository>
-			<id>ossrh</id>
-			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
-		</snapshotRepository>
-		<repository>
-			<id>ossrh</id>
-			<url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-		</repository>
-	</distributionManagement>
-
-	<dependencies>
-		<dependency>
-			<groupId>org.abego.treelayout</groupId>
-			<artifactId>org.abego.treelayout.core</artifactId>
-			<version>1.0.1</version>
-		</dependency>
-
-	</dependencies>
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.antlr</groupId>
+    <artifactId>antlr4-master</artifactId>
+    <version>4.5.1-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+  <artifactId>antlr4-runtime</artifactId>
+  <name>ANTLR 4 Runtime</name>
+  <description>The ANTLR 4 Runtime</description>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.abego.treelayout</groupId>
+      <artifactId>org.abego.treelayout.core</artifactId>
+      <version>1.0.1</version>
+    </dependency>
+  </dependencies>
 
+  <build>
+    <sourceDirectory>src</sourceDirectory>
+    <plugins>
+      <plugin>
+        <groupId>org.antlr</groupId>
+        <artifactId>antlr4-maven-plugin</artifactId>
+        <version>4.5</version>
+        <executions>
+          <execution>
+            <id>antlr</id>
+            <configuration>
+              <sourceDirectory>src</sourceDirectory>
+            </configuration>
+            <goals>
+              <goal>antlr4</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>   
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>2.5.4</version>
+        <executions>
+          <execution>
+            <id>bundle-manifest</id>
+            <phase>process-classes</phase>
+            <configuration>            
+              <instructions>
+                <Bundle-SymbolicName>org.antlr.antlr4-runtime-osgi</Bundle-SymbolicName>
```

Comment:
:thought_balloon: These properties don't match the ones being produced by bild.py. The impact of the difference is not clear to me.